lib/getWorkflow.js

```diff
@@ -75,7 +77,20 @@ const calculateNodes = async (jobs, triggerFactory, pipelineId) => {
             externalDownstreamAnd, nodes, triggerFactory);
     }));
 
-    return [...nodes].map(name => ({ name }));
+    return [...nodes].map((name) => {
+        const m = { name };
+        const displayName = hoek.reach(
+            jobs[name],
+            'annotations>screwdriver.cd/displayName',
+            { separator: '>', default: undefined }
+        );
+
+        if (displayName !== undefined) {
+            m.displayName = displayName;
```

Comment:
do we need to update with `displayName` for workflowGraph? https://github.com/screwdriver-cd/data-schema/blob/master/config/workflowGraph.js

Reply:
Oh, that specifies `unknown()`, it seems to work without adding it, but it's more appropriate to add it.
I tried a bit, but I can't figure out how to add a `displayName` as an option, and I get a validation error of `pipelines[0].workflowGraph.nodes[3].displayName" must be a string` 🤔

Reply:
Hm this doesn't work?
https://github.com/screwdriver-cd/data-schema/compare/displayName?expand=1

Reply:
That does not work.
```
    nodes: Joi.array().items(Joi.object().keys({
        name: requiresValue,
        displayName: jobName
    }).unknown()),
```

It didn't work even with the `optional()` as shown below.
```
    nodes: Joi.array().items(Joi.object().keys({
        name: requiresValue,
        displayName: jobName.optional()
    }).unknown()),
```

It seems that an error occurs if the displayName key is omitted.

Reply:
wonder if it's because the `unknown()` keyword
okay with merging this PR as-is for now and figuring out how to add `displayName` to the `workflowGraph` later

Reply:
OK, I've tried everything, but it's still not working. I will do so. Thanks!

Reply:
Ah, I've found the cause. It happens when I set the `screwdriver.ce/displayName` to a number.
```
jobs:
  main:
    steps:
      - ls
    image: node:12
    annotations:
      screwdriver.cd/displayName: 0
```

Checking with [`SCHEMA_WORKFLOW_GRAPH`](https://github.com/screwdriver-cd/data-schema/blob/master/config/workflowGraph.js) will result in a 404 error on the pipeline page, so validation step of yaml should return an error. I will consider validating the value of annotations.